src/main.rs

```diff
@@ -15,127 +16,119 @@ use action::Action;
-                .long("verbose")
-                .default_value("error")
-                .takes_value(true),
-        )
-        .arg(
-            Arg::with_name("arch")
-                .help("Set the architecture to select in fat binaries")
-                .short('a')
-                .long("arch")
-                .default_value(common::get_compile_time_arch())
-                .takes_value(true),
-        )
-        .arg(
-            Arg::with_name("type")
-                .help("Ignored, listed for compatibility only")
-                .short('t')
-                .long("type")
-                .default_value("")
-                .takes_value(true),
-        ).arg(
-            Arg::with_name("list_arch")
-                .help("List the architectures present in the fat binaries")
-                .long("list-arch")
-        )
-        .arg(
-            Arg::with_name("num_jobs")
-                .help("Number of jobs")
-                .short('j')
-                .value_name("NUMBER")
-                .default_value("")
-                .takes_value(true),
-        )
-        .arg(
-            Arg::with_name("mapping_var")
-                .help("A pair var=value such as rev=123abcd")
-                .long("mapping-var")
-                .multiple(true)
-                .takes_value(true),
-        )
-        .arg(
-            Arg::with_name("mapping_src")
-                .help("Regex to match a path with capturing groups")
-                .long("mapping-src")
-                .multiple(true)
-                .takes_value(true),
-        )
-        .arg(
-            Arg::with_name("mapping_dest")
-                .help(r#"A replacement string using groups, variables (set with --mapping-var), special variable like DIGEST or digest.
+fn cli() -> Command {
+    Command::new("dump_syms")
+    .version(crate_version!())
+    .author(crate_authors!("\n"))
+    .about("Dump debug symbols to breakpad symbols")
+    .arg(
+        Arg::new("filenames")
+            .help("Files to dump (.dll, .exe, .pdb, .pd_, .so, .dbg)")
+            .required(true)
+            .num_args(1..)
+    )
+    .arg(
+        Arg::new("output")
+            .help("Output file or - for stdout")
+            .short('o')
+            .long("output")
+    )
+    .arg(
+        Arg::new("store")
+            .help("Store output file as FILENAME.pdb/DEBUG_ID/FILENAME.sym in the given directory")
+            .short('s')
+            .long("store")
+    )
+    .arg(
+        Arg::new("debug_id")
+            .help("Get the pdb file passed as argument from the cache or from symbol server using the debug id")
+            .long("debug-id")
+    )
+    .arg(
+        Arg::new("code_id")
+            .help("Get the dll/exe file passed as argument from the cache or from symbol server using the code id")
+            .long("code-id")
+    )
+    .arg(
+        Arg::new("symbol-server")
```

Comment:
I noticed that for all the other values we use an underscore vs a dash in the option name, maybe we can change this one too for consistency?